smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -258,7 +259,7 @@ def manualRefresh() {
 }
 
 def uninstalled(){
-	atomicState.bridges = [:]
```

Comment:
Clearing the state in the uninstalled() method is moot. There's no way to ever get to it again. Shouldn't hurt though